.github/actions/publish-npm/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish to NPM
+description: Publish an npm package.
+inputs:
+  prerelease:
+    description: 'Is this a prerelease. If so, then the latest tag will not be updated in npm.'
+    required: true
+  dry_run:
+    description: 'Is this a dry run. If so no package will be published.'
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Publish
+      shell: bash
+      run: |
+        ./scripts/publish-npm.sh
+      env:
+        LD_RELEASE_IS_PRERELEASE: ${{ inputs.prerelease }}
```

Comment:
What changes when prerelease is set to `true`?

Reply:
It would be for tagging the npm package so we don't bump the normal version. We probably won't need to use it.

Reply:
If you were going to be doing beta/alpha versions, then it would potentially be something you would want. (Or even nightlies 0.7.0-nightly-somesha). 

Depending on how you are going to manage releases it may not be relevant.